abixen-platform-web-client/src/main/web/admin/scss/application.scss

```diff
@@ -65,6 +65,14 @@ body {
 
 .platform-navigation {
   @include platform-navigation($font-color, $base-color-darkest, $base-color-darker, $complementary-color);
+  .top-bar .user>.item img {
+   -webkit-border-radius: 50%;
```

Comment:
We have the function @mixin border-radius($radius) - please use it.
For border colour please use:

border-style: solid;
border-color: $base-color;

We have already defined a list of colours variables we have to use it.